src/bot/scripts/card.js

```diff
@@ -728,946 +66,1233 @@ const card = new Command({
+
+                // ***ATTACK INFO***
+
+                // -can attack
+                // -attack range
+                // -time in between attacks
+                // -pre attack delay
+                // -knockback impulse
+                // -aoe attack type
+                // -aoe damage percentage
+                // -aoe radius
+                // -aoe knockback percentage
+
+                // ***POWER INFO***
+
+                // -targeting ...
+                // -power duration
+
+                // ***SPEED INFO***
+
+                // -time to max velocity
+                // -max velocity
+
+                // ***totem ONLY***
+
+                // -health loss
+
+                // ***spell ONLY***
+
+                // *** ??? ***
+
+                // -requirements ...
+                // -child unit limit
+
+                // *name
+                // CanAttack
+                // *description
+                // *Image
+                // -ManaCost
+                // -damage
+                // -health
+                // healthLoss - character_type: totem
+                // type - if !Character, character_type === undefined
+                // Targeting ... - power radius
+                // character_type
+                // AttackRange
+                // TimeToMaxVelocity
+                // MaxVelocity
+                // TimeInBetweenAttacks
+                // PowerDuration
+                // -PowerPower
+                // -rarity
+                // -theme
+                // Requirements
+                // AOEAttacktype
+                // AOEdamagePercentage
+                // AOERadius
+                // AOEKnockbackPercentage
+                // PreAttackDelay
+                // CastArea - ownside/anywhere
+                // ChildUnitLimit
+
+                await message.channel.send("", {
+                  file: path.join(__dirname, "temp", `pd-${saveDate}.png`)
+                });
+                message.channel.send(embed);
+                fs.unlink(
+                  path.join(__dirname, "temp", `pd-${saveDate}.png`),
+                  error => {
+                    if (error !== null && error !== undefined) {
+                      throw `could not delete: pd-${saveDate}.png`;
+                    }
+                  }
+                );
+              }
+            );
+          })
+          .catch(error => {
+            console.error(error);
+            const discordEmbed = new discord.RichEmbed()
+              .setColor("#F8534F")
+              .setTitle(":x: Error 1. Please contact the developer");
+            message.channel.send(discordEmbed);
+          });
+      })
+      .catch(error => {
+        console.error(error);
+        const discordEmbed = new discord.RichEmbed()
+          .setColor("#F8534F")
+          .setTitle(":x: Error 2. Please contact the developer");
+        message.channel.send(discordEmbed);
       });
-    });
   },
 
   load: function () {
```

Comment:
ik this was here before, but do the same as cb; define it below and pass it here, also should be safe to make it an arrow fn, but do check if it all works afterwards

Reply:
how do i make that an arrow function

Reply:
how do i even call the const?

Reply:
same as the 'cb' one above